engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -83,6 +83,14 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryWorkflowOutputsAndRespond(id: WorkflowId): Unit = {
+    val replyTo = sender()
+    dataAccess.queryWorkflowOutputs(id) onComplete {
+      case Success(o) => replyTo ! WorkflowOutputsResponse(id, o)
+      case Failure(t) => replyTo ! WorkflowOutputsFailure(id, t)
```

Comment:
Actually after further thought and discussion with @cjllanwarne I'll just ticket this rather than fix it as part of this PR.  I'm not sure we currently have enough information in the PBE Cromwell DB to definitively know whether a workflow exists or not.  It's not necessarily correct to infer from the absence of workflow outputs that a workflow doesn't exist; it's possible the outputs haven't been written yet, or a given workflow is all side effects and has no formally declared outputs.  Perhaps we might infer that the presence of *any* metadata for a workflow ID should imply the existence of the workflow, but given the fully fire-and-forget nature of metadata writes I don't think that's quite the contract we have on the metadata journal at the moment.